src/test/java/uk/gov/justice/probation/courtcasematcher/model/mapper/CaseMapperTest.java

```diff
@@ -109,46 +113,83 @@ void whenMapNewCaseThenCreateNewCaseNoOffences() {
         assertThat(courtCase.getOffences()).isEmpty();
     }
 
-    @DisplayName("Map a new case from gateway case but with no offences")
+    @DisplayName("Map a court case to a new court case when search response has yielded no matches")
     @Test
-    void whenMapNewFromCaseAndOffender_thenCreateNewCaseWithOffenderData() {
+    void givenNoMatches_whenMapNewFromCaseAndSearchResponse_thenCreateNewCaseWithEmptyListOfMatches() {
+        final String crn2 = "X340987";
 
-        ReflectionTestUtils.setField(aCase, "offences", null);
-        GroupedOffenderMatches matches = GroupedOffenderMatches.builder()
-            .matches(singletonList(
-                        OffenderMatch.builder()
-                            .matchType(MatchType.NAME_DOB)
-                            .build()))
+        Match match1 = Match.builder().offender(Offender.builder()
+                                                    .otherIds(OtherIds.builder()
+                                                                        .crn(CRN)
+                                                                        .build())
+                                                    .build())
+                                                .build();
+        Match match2 = Match.builder().offender(Offender.builder().otherIds(OtherIds.builder().crn(crn2).build()).build()).build();
+
+        CourtCase courtCase = caseMapper.newFromCase(aCase);
+        SearchResponse searchResponse = SearchResponse.builder()
+            .matchedBy(OffenderSearchMatchType.ALL_SUPPLIED)
```

Comment:
Doubt it will make any functional difference but I think it would return `OffenderSearchMatchType.NOTHING` in this case

Reply:
Done